queue_services/entity-emailer/src/entity_emailer/worker.py

```diff
@@ -57,15 +57,14 @@
     restoration_notification,
 )
 
-from .message_tracker import tracker as tracker_util
-
 
 qsm = QueueServiceManager()  # pylint: disable=invalid-name
 APP_CONFIG = config.get_named_config(os.getenv('DEPLOYMENT_ENV', 'production'))
 FLASK_APP = Flask(__name__)
 FLASK_APP.config.from_object(APP_CONFIG)
 db.init_app(FLASK_APP)
 
+from .message_tracker import tracker as tracker_util
```

Comment:
I'm not sure whether it's proper to change the order of import here. ❔  But it can avoid repeated importing and partial import error.

Reply:
Oops. It will make linter fail. 😨